src/public_keys.h

```diff
@@ -105,6 +105,31 @@ static const uint8_t LEDGER_NFT_SELECTOR_PUBLIC_KEY[] = {
 #endif
 };
 
+static const uint8_t DYNAMIC_NETWORK_PUB_KEY[] = {
```

Comment:
If this uses the CAL key, might as well reuse it instead of duplicating it.

Reply:
Removed